janusgraph-hadoop/src/main/java/org/janusgraph/hadoop/ImmutableConfiguration.java

```diff
@@ -1,442 +0,0 @@
-// Copyright 2017 JanusGraph Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package org.janusgraph.hadoop;
-
-import org.apache.hadoop.classification.InterfaceStability;
-import org.apache.hadoop.conf.Configuration;
-import org.apache.hadoop.fs.Path;
-
-import java.io.DataInput;
-import java.io.DataOutput;
-import java.io.File;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
-import java.io.Reader;
-import java.io.Writer;
-import java.net.InetSocketAddress;
-import java.net.URL;
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-import java.util.concurrent.TimeUnit;
-import java.util.regex.Pattern;
-
-public class ImmutableConfiguration extends Configuration {
```

Comment:
Shall we mark this class as deprecated first, and then remove it in the next release?

Reply:
I could deprecated it in 0.6.1. What do you think?

First of all, i will remove it from this pr.

Reply:
Deprecating it in 0.6.0 sounds good. I guess you could keep the change in this PR then, since it's targeting the main branch.

Reply:
I would do the deprecation as CTR. If it is okay for you.

Reply:
Done

Reply:
Should this file also be added to the changelog?

Reply:
file is added